### PR TITLE
Add `activity` subcommand and template

### PR DESCRIPTION
**Pull request by t-eckert**
## What is the goal of this pull request?

Add CLI command `cargo func new activity`.

## What does this pull request change?

Adds a template for Activity Functions, adds subcommand for creating a new Activity Function.

## What work remains to be done?

Check that the Activity Function template is understandable

## Do you consider it adequately tested?

YES, I was able to run the command and create the correct templated function.

### CODE REVIEW

File: azure-functions-sdk/src/templates/new/activity.rs.hbs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+use azure_functions::{bindings::DurableActivityContext, func};
+
+#[func]
+pub async fn {{name}}(context: DurableActivityContext) {
+    context.set_output("Hello from {{name}}!");
+}
```
Comment (peterhuene) [inline azure-functions-sdk/src/templates/new/activity.rs.hbs]: Nit: I think the other templates have a trailing newline.  Let's add one here too.

File: azure-functions-sdk/src/commands/new.rs [hunk 2]
```diff
@@ -666,3 +668,38 @@ impl<'a> From<&'a ArgMatches<'a>> for ServiceBus<'a> {
...
+    }
+}
+
+impl<'a> From<&'a ArgMatches<'a>> for Activity<'a> {
+    fn from(args: &'a ArgMatches<'a>) -> Self {
+        Activity {
+            name: args.value_of("name").unwrap(),
+        }
+    }
+}
```
Comment (peterhuene) [inline azure-functions-sdk/src/commands/new.rs]: The missing trailing newline here is the source of the formatting diff.

**Comment by peterhuene** [comment]
We should push up a commit to delete the `bindings::durable_orchestration_context::tests::it_constructs` test for now.  It was just a stub for a non-functional implementation anyway.

File: azure-functions/src/bindings/durable_activity_context.rs [hunk 3]
```diff
@@ -33,6 +33,7 @@ mod tests {
     use crate::rpc::typed_data::Data;
 
     #[test]
+    #[ignore]
```
Comment (peterhuene) [inline azure-functions/src/bindings/durable_activity_context.rs]: Wrong file.  Should be the `durable_orchestration_context.rs`.

Comment (peterhuene) [reply azure-functions/src/bindings/durable_activity_context.rs]: This test is fine as we don't parse any data yet.

Comment (t-eckert) [reply azure-functions/src/bindings/durable_activity_context.rs]: Oops

**Comment by t-eckert** [comment]
Ok to merge? @peterhuene

**Comment by peterhuene** [comment]
Will merge when the night Windows CI completes.

**Pull request merged**